Neos.ContentRepository.Core/Classes/SharedModel/ContentRepositoryNodeInformationInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Neos\ContentRepository\Core\SharedModel;
+
+use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamId;
+use Neos\ContentRepository\Core\SharedModel\Node\NodeAggregateId;
+
+interface ContentRepositoryNodeInformationInterface
```

Comment:
Uhm, I woudn't create interfaces here which are not implemented by the core.

Reply:
I wonder if we could just omit the interface for now. Seems like a premature generalization to me at this point

Reply:
Is removed and "replaced" by `Neos\Neos\AssetUsage\Dto\AssetUsageReference`